com.unity.multiplayer.mlapi/Runtime/Core/MessageBatcher.cs

```diff
@@ -187,31 +187,34 @@ public void SendItems(int threshold, SendCallbackType sendCallback)
         /// <param name="receiveTime"> the packet receive time to pass back to callback</param>
         public int ReceiveItems(in BitStream messageStream, ReceiveCallbackType receiveCallback, MLAPI.RpcQueueContainer.QueueItemType messageType, ulong clientId, float receiveTime)
         {
-            using PooledBitStream copy = PooledBitStream.Get();
-            do
+            using (var copy = PooledBitStream.Get())
```

Comment:
this using declaration is C# 8.0 which is not supported on 2019.4 and 2020.1 @jeffreyrainy :)
but using (...) { ... } block is fine!